docs/two-phase-commit-transactions.md

```diff
@@ -5,6 +5,21 @@ With Two-phase Commit Transactions, you can execute a transaction that spans mul
 
 This document briefly explains how to execute Two-phase Commit Transactions in ScalarDB.
 
+## Overview
+
+ScalarDB transactions normally execute in a single transaction manager instance.
+In that case, you begin a transaction, execute CRUD operations, and commit the transaction in the same transaction manager instance.
+
+In addition to normal transactions, ScalarDB also supports two-phase-commit-style transactions called *Two-phase Commit Transactions*.
```

Comment:
Sorry, I may have capitalized `Two-phase Commit Transactions` since I thought it was referring to the name of the feature in ScalarDB. If this is a generic mention of `two-phase commit transactions`, let's lowercase it. 

And I would keep `normal transactions` lowercase because, as far as I know, it's not the name of a feature in ScalarDB.